### DISCUSSION TITLE
Does it support WebP on mac builds ?

### DISCUSSION THREAD

**Discussion by capellavacancies** [PintaProject/Pinta#769] [General]
I've read online WebP it's supported ([source](https://www.html.it/magazine/pinta-2-1-implementato-il-supporto-a-webp-e-migliorato-quello-per-wayland/)) but when I try to open a WebP file it says it's not supported. Ideas ?

**Comment by cameronwhite** [comment]
This indeed should work, but something seems broken .. I'll convert this to a bug
Submitted as bug #770
